src/test/objectstore/allocator_replay_test.cc

```diff
@@ -5,20 +5,22 @@
  * Author: Igor Fedotov, ifedotov@suse.com
  */
 #include <iostream>
+#include <vector>
 
 #include "common/ceph_argparse.h"
 #include "common/debug.h"
 #include "common/Cycles.h"
 #include "common/errno.h"
 #include "common/ceph_json.h"
 #include "common/admin_socket.h"
+#include "include/denc.h"
 #include "global/global_init.h"
 #include "os/bluestore/Allocator.h"
 
 using namespace std;
 
 void usage(const string &name) {
-  cerr << "Usage: " << name << " <log_to_replay> <raw_duplicate|free_dump|try_alloc count want alloc_unit|replay_alloc alloc_list_file>" << std::endl;
+  cerr << "Usage: " << name << " <log_to_replay> <raw_duplicate|free_dump|try_alloc count want alloc_unit|replay_alloc alloc_list_file|export_binary out_file>" << std::endl;
 }
```

Comment:
How can one use that 'export_binary' data?

Reply:
The replay tool can use dumps in this format on par with the json format. But this new format drastically increases dump loading time making it almost instant. Which is nice when you're doing some troubleshooting and need to run the replay tool multiple times